src/services/localizations/assertValidLanguageCode.ts

```diff
@@ -11,8 +11,8 @@ export const availableLanguagesCodes = [
   'ZU',
 ];
 
-export const assertValidLanguageCode = (languageCode: string) => {
+export function assertValidLanguageCode(languageCode: string) {
   if (!availableLanguagesCodes.includes(languageCode)) {
-    throw new Error(`The language code ${languageCode} is not available! The available language codes are [${availableLanguagesCodes.join(', ')}]`);
+    throw new Error(`The language code ${languageCode} is not available! The available language codes are: ${availableLanguagesCodes.join(', ')}`);
```

Comment:
(Question) Would it make sense that rather than throwing an error here we show a warning and don't synchronize invalid languages?

Otherwise the process would exit right and you could have synced like 5/10 languages correctly?

Reply:
I agree if it would result in a partial sync, but it first validates everything before it starts the sync. I feel like have a faulty language file would be reason enough for an error, feels weird to continue syncing the rest 🤔